db/migrate/20171115191926_rename_employed_monthly_employer_to_employer_employer_names.rb

```diff
@@ -0,0 +1,41 @@
+class RenameEmployedMonthlyEmployerToEmployerEmployerNames < ActiveRecord::Migration[5.1]
+  def up
+    add_column :members, :employed_employer_names, :string, array: true
```

Comment:
renamed these fields to make their meaning clearer.

Just had a call with @rachelcope where I confirmed that in the next iteration of the "amounts - income" page, we are going to ask for employment pay quantities and intervals, so nothing related to monthly income

Reply:
This feels like an improvement, but I still find it a little confusing to read `employed_employer`. would it be clearer if it were `job_employer_names` or `employment_employer_names`? 
It seems like a noun in place of the `employed` adjective would make it clear that this is a property of a thing, which I think is implied.

Reply:
yeahhhh -- the reason I named it this way was that we prepend ever `job` field with `employed` and every self employment field with `self_employed` -- I am not sure this pattern is even that helpful at this point, but it is an existing pattern that provides some means of remembering which fields are named what. Do you think we should move away from that pattern, or perhaps do that when moving this data to a separate table?